components/eamxx/src/physics/rrtmgp/CMakeLists.txt

```diff
@@ -59,8 +67,13 @@ set(EXTERNAL_SRC
 )
 add_library(rrtmgp ${EXTERNAL_SRC})
 EkatDisableAllWarning(rrtmgp)
-SetCudaFlags(rrtmgp)
-target_link_libraries(rrtmgp PUBLIC yakl)
+SetCudaFlags(rrtmgp CUDA_LANG)
```

Comment:
Is this necessary if we are using `yakl_process_target` below?

Reply:
I think so. In debug mode, we add `--fmad=false` with EKAT, but YAKL doesn't. We could just add `--fmad=false` to `YAKL_CUDA_FLAGS` if it's a debug build, I guess.